### PR TITLE
adding reactions

### PR DESCRIPTION
**Pull request by eliotanke**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -15,122 +15,145 @@ const base_slack_url = 'https://slack.com/api';
 const team_name = 'elio.tanke';
 const table_name = 'github_pr_slack_message_mapping';
 
-const username_email_map = {
-    'rosalynn-chong-clio': 'rosalynn.chong@clio.com',
-    'coryb2424': 'cory.bonneau@clio.com',
-    'JRSpencer': 'james.spencer@clio.com',
-    'rogerli2012': 'roger.li@clio.com',
-    'eliotanke': 'elio.tanke@clio.com',
+const github_slack_username_map = {
```
Comment (eliotanke) [inline index.js:18]: testing123

Comment (eliotanke) [inline index.js:18]: another comment

File: index.js [hunk 2]
```diff
@@ -15,122 +15,145 @@ const base_slack_url = 'https://slack.com/api';
...
+    'rosalynn-chong-clio': 'rosalynn.chong',
```
Comment (eliotanke) [inline index.js:19]: aaa

File: index.js [hunk 3]
```diff
@@ -15,122 +15,145 @@ const base_slack_url = 'https://slack.com/api';
...
+    'coryb2424': 'cory.bonneau',
```
Comment (eliotanke) [inline index.js:20]: testing123

**Review by elio303 — APPROVED** [review]

**Review by elio303 — COMMENTED** [review]
aaa

**Review by elio303 — COMMENTED** [review]
aaaa

**Review by elio303 — COMMENTED** [review]
aaaaa

**Review by elio303 — APPROVED** [review]
aaaa

File: index.js (hunk 1 above)
Comment (elio303) [inline index.js:18]: Comment

Comment (elio303) [inline index.js:18]: Comment

File: index.js [hunk 4]
```diff
@@ -15,122 +15,145 @@ const base_slack_url = 'https://slack.com/api';
...
+    'JRSpencer': 'james.spencer',
+    'rogerli2012': 'roger.li',
+    'eliotanke': 'elio.tanke',
 };
 
-// ---------- Axios wrappers to call the Slack API
+// ---------- Simple helpers
 
-const get = async (url, data) => await axios.post(url, qs.stringify(data));
+const getCreateMessage = (url, title) => `@${team_name} CR please: <${url}|${title}>`;
```
Comment (elio303) [inline index.js:28]: comment

**Review by elio303 — COMMENTED** [review]
aaa

**Review by elio303 — COMMENTED** [review]
sfsfdsfsfa

**Review by elio303 — CHANGES_REQUESTED** [review]
changes

**Review by elio303 — COMMENTED** [review]
comment1

**Review by elio303 — COMMENTED** [review]
comment1

File: index.js [hunk 5]
```diff
@@ -15,122 +15,145 @@ const base_slack_url = 'https://slack.com/api';
...
-        form,
-        { headers: form.getHeaders() }
-    );
+    Object.entries(data).forEach(([key, value]) => form.append(key, value));
+    return form;
+};
+
+const get = async (path, data) => await axios.post(`${base_slack_url}/${path}`, qs.stringify(data));
+
+const post = async (path, data) => {
```
Comment (elio303) [inline index.js:43]: comment 2

File: index.js [hunk 6]
```diff
@@ -15,122 +15,145 @@ const base_slack_url = 'https://slack.com/api';
...
+    const url = `${base_slack_url}/${path}`;
+    const form = createForm(data);
+
```
Comment (elio303) [inline index.js:46]: comment 3

**Review by elio303 — COMMENTED** [review]
comment 4

File: index.js (hunk 1 above)
Comment (elio303) [inline index.js:18]: comment 5

**Review by elio303 — APPROVED** [review]
comment 6

**Review by elio303 — CHANGES_REQUESTED** [review]
comment 7

**Pull request merged**